ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -91,6 +81,20 @@ public void close() {
     commandStore.close();
   }
 
+  public void fetchAndRunCommands() {
+    ConsumerRecords<CommandId, Command> records = commandStore.getNewCommands();
+    log.debug("Found {} new writes to command topic", records.count());
+    for (ConsumerRecord<CommandId, Command> record : records) {
+      CommandId commandId = record.key();
+      Command command = record.value();
+      if (command.getStatement() != null) {
+        executeStatement(command, commandId);
+      } else {
+        log.debug("Skipping null statement for ID {}", commandId);
```

Comment:
In what circumstance does this happen?

Reply:
Technically we should not have any command with null value since KSQL should be the only one who writes the commands to the command topic. I guess this remained here when our intern was refactoring. I'll remove it.